portfolio/src/main/webapp/index.html

```diff
@@ -1,39 +1,187 @@
+                                    <p>Gourmonade <br>San Francisco, CA</p>
+                                </div>
+                            </div>		
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food6.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Two Hands <br>Little Italy, NY</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food3.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Home <br>San Francico, CA</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food4.jpg" alt="Anika's profile picture" />
+                            <div class="caption">
+                                <div class="blur"></div>
+                                <div class="caption-text">
+                                    <p>Ippodu <br>San Francico, CA</p>
+                                </div>
+                            </div>	
+                        </div>
+                    </div>   
+                </div>
+            </div>
+
+            <div class="row food-second">
+                <div class="caption-style-1">
+                    <div class="col-md-3">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food1-new.jpg" alt="Anika's profile picture" />
+                                <div class="caption">
+                                    <div class="blur"></div>
+                                    <div class="caption-text">
+                                        <p>Old Town Mexican Cafe<br>San Diego, CA</p>
+                                    </div>
+                                </div>	
+                        </div>
+                    </div>
+                    <div class="col-md-3 ">
+                        <div class="food-img">
+                            <img style="width: 90%; height: 25%" src="images/food2.jpg" alt="Anika's profile picture" />
```

Comment:
since this style is repeated, it's a good candidate for a class in your style.css file

```
.food-image {
  width: 90%;
  height: 25;
}
```

Reply:
fixed